locales/en.yml

```diff
@@ -153,3 +153,6 @@ en:
         status:
           running: running
           stopped: stopped
+        unsupported_service: |-
+          Service '%{service}' is not supported.
```

Comment:
I'll say something like "Unkown service '%{service}'". That makes imo more sense in the case of a typo.